src/Utils/fetchUserPreferences.ts

```diff
@@ -0,0 +1,18 @@
+import { Environment, fetchQuery, graphql } from "relay-runtime"
+import { fetchUserPreferencesQuery } from "__generated__/fetchUserPreferencesQuery.graphql"
+
+export const fetchUserPreferences = (relayEnvironment: Environment) => {
+  return fetchQuery<fetchUserPreferencesQuery>(
+    relayEnvironment,
+    USER_PREFERENCES_QUERY,
+    {}
+  )
+}
+
+const USER_PREFERENCES_QUERY = graphql`
+  query fetchUserPreferencesQuery {
+    me {
+      lengthUnitPreference
+    }
+  }
+`
```

Comment:
This file might be more appropriately placed in the `/System` folder, close to where its used (and would be modified later)

Reply:
do you mean to place this file under the path `src/System/fetchUserPreferences.ts`?

Reply:
done